controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -127,6 +131,10 @@ func (r *StorageConsumerReconciler) initReconciler(request reconcile.Request) {
 	r.cephClientHealthChecker = &rookCephv1.CephClient{}
 	r.cephClientHealthChecker.Name = GenerateHashForCephClient(r.storageConsumer.Name, "global")
 	r.cephClientHealthChecker.Namespace = r.namespace
+
+	r.noobaaAccount = &nbv1.NooBaaAccount{}
+	r.noobaaAccount.Name = "noobaa-remote-" + r.storageConsumer.Status.Client.ClusterID
```

Comment:
there is a subtle bug here, the status will be available here only after atleast a single heartbeat. you may need to set this based on the presence of the ID during reconcile directly.